dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -111,12 +150,12 @@ internal constructor(
             NoOpRumResourceAttributesProvider(),
         @FloatRange(from = 0.0, to = 100.0) traceSamplingRate: Float = DEFAULT_TRACE_SAMPLING_RATE
     ) : this(
-        tracedHosts = firstPartyHosts,
+        tracedHosts = firstPartyHosts.associateWith { setOf(TracingHeaderType.DATADOG) },
         tracedRequestListener = tracedRequestListener,
-        firstPartyHostDetector = getGlobalFirstPartyHostDetector(),
+        firstPartyHostResolver = getGlobalFirstPartyHostDetector(),
```

Comment:
Should we change the name of the method to `getGlobalFirstPartyResolver` to match the property name change?